docs/intro/from-ground-up.rst

```diff
@@ -178,9 +178,9 @@ The same, but using web-poet
 
 Differences from a previous example:
 
-* instead of dicts with "url" and "text" fields, :class:`~.ResponseData`
-  instances are used. :class:`~.ResponseData` is a simple structure with
-  two fields ("url" and "html"), defined by web-poet;
+* instead of dicts with "url" and "text" fields, :class:`~.HttpResponse`
+  instances are used. :class:`~.HttpResponse` is a simple structure with
+  two fields ("url" and "body"), defined by web-poet;
```

Comment:
It has more fields now. Wee should check the whole from-ground-up tutorial; it has some inconsistencies after the removal of ResponseData.